packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.tsx

```diff
@@ -337,21 +342,21 @@ const FinancialAssessment = observer(() => {
 
     const getScrollOffset = () => {
         if (is_mf) {
-            if (isMobile() && is_financial_information_incomplete) return '220px';
-            return is_financial_information_incomplete && !is_submit_success ? '165px' : '160px';
-        } else if (isMobile()) return is_appstore ? '160px' : '200px';
-        return '80px';
+            if (is_mobile && is_financial_information_incomplete) return '22rem';
+            return is_financial_information_incomplete && !is_submit_success ? '16.5rem' : '16rem';
+        } else if (is_mobile) return '20rem';
+        return '8rem';
     };
```

Comment:
`if (is_mf) {
    if (is_mobile ) {
        if (is_financial_information_incomplete) {
            return !is_submit_success ? '16.5rem' : '16rem'
        }
        return '20rem'
    }
return '8rem';
};`

would it be more readable?
you can ignore  this :)